components/sup-protocol/protocols/ctl.proto

```diff
@@ -104,6 +104,8 @@ message SvcLoad {
   optional sup.types.Topology topology = 12;
   // Update strategy for the service.
   optional sup.types.UpdateStrategy update_strategy = 13;
+  // only render package templates
+  optional bool render_only = 15 [default = false];
```

Comment:
Why not 14?

Reply:
`14` is already used by `binding_mode` above.

Reply:
🤦‍♂️  Maybe we should put these in order.